core/runtime/src/main/java/io/quarkus/logging/LoggingFilter.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.logging;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Makes the filter class known to Quarkus by the specified name.
+ * The filter can then be configured for a handler (like the logging handler using {@code quarkus.log.console.filter}).
+ *
+ * This class must ONLY be placed on implementations of {@link java.util.logging.Filter} that are marked as {@code final}.
```

Comment:
Out of curiosity: Why the `Filter` implementation class must be final?

Reply:
It just makes it easier to ensure that there is ambiguity if subclasses are involved. We don't strictly need it, we can certainly relax this if people find themselves needing to subclass